config/prod/minidream-rstudio-2021-ec2-alb.yaml

```diff
@@ -0,0 +1,22 @@
+# Provision a public load balancer in front of private miniDREAM EC2
+# Based on prod/minidream-R-env-pub-prod-instance-ec2-alb-v2.yaml
+template_path: "remote/managed-alb-https-v2.yaml"
+stack_name: "minidream-rstudio-2021-ec2-alb"
+
+parameters:
+  Department: "CompOnc"
+  Project: "CSBC"
+  OwnerEmail: "bruno.grande@sagebase.org"
+  Ec2InstanceId: !stack_output "prod/minidream-rstudio-2021-ec2.yaml::Ec2InstanceId"
```

Comment:
this depends on the deployment of `config/prod/minidream-rstudio-2021-ec2.yaml`.  i think you need to add a `dependencies` to this file so that this template will get deployed after the EC2 template.  Something like this.. https://github.com/Sage-Bionetworks/scicomp-provisioner/blob/a7a43e4b4221bba40a0e6dd4dacfad4464e140f6/config/prod/htan-dcc-dfci.yaml#L12

Reply:
![image](https://user-images.githubusercontent.com/740725/121095176-add65180-c7a4-11eb-9cc8-3edc8fb8f0c3.png)